### PR TITLE
Update link and validate example

### PR DESCRIPTION
**Pull request by taysea**
- Updates text input link to use master file
- Updates TextArea validation example to match how TextInput/Select work

### CODE REVIEW

File: aries-site/src/pages/components/textinput.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const TextInput = () => (
       <Subsection name={title} level={1} topic={topic}>
         <SubsectionText>{page.description}</SubsectionText>
         <Example
-          designer="https://designer.grommet.io/textinput?id=HPE-design-system-taylor-seamans-hpe-com&mode=edit"
```
Comment (britt6612) [inline aries-site/src/pages/components/textinput.js:31]: should we switch out links ? for the main designer

**Comment by britt6612** [comment]
Looks good thanks for switching this out!

File: aries-site/src/pages/components/textinput.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/pages/components/textinput.js:31]: It should be updated

Comment (britt6612) [reply aries-site/src/pages/components/textinput.js:31]: okay good to know!

**Pull request merged**